travis-ymls/sort-package-json.travis.yml

```diff
@@ -0,0 +1,26 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : sort-package-json
+# Source Repo         : https://github.com/keithamus/sort-package-json.git
+# Travis Job Link     : https://travis-ci.com/github/nagesh4193/sort-package-json/builds/215038463
+# Created travis.yml  : Yes
+# Maintainer          : Nageswara Rao K<nagesh4193@gmail.com>/Priya Seth<sethp@us.ibm.com>
+#
+# Script License      : Apache License, Version 2 or later
+#
+# ----------------------------------------------------------------------------
+
+language: node_js
+
+node_js:
+  - 12
+  - 13
+  - 14
+  - 15
```

Comment:
Add latest node versions as mentione in `setimmediate`

Reply:
Updated with latest versions of nodes.